src/core/MOM_barotropic.F90

```diff
@@ -2054,7 +2103,7 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
       do J=jsv-1,jev ; do i=isv-1,iev+1
         vel_prev = vbt(i,J)
         vbt(i,J) = bt_rem_v(i,J) * (vbt(i,J) + &
-             dtbt * ((BT_force_v(i,J) + Cor_v(i,J)) + PFv(i,J)))
+             dtbt * ((BT_force_v(i,J) + Cor_v(i,J)) + PFv(i,J) - Drag_v(i,J)))
```

Comment:
As written, `Drag_u` and `Drag_v` do not appear to be changing within the barotropic time-stepping loops, so is there a reason why they could not be included in `BT_force_u` and `BT_force_v`?

Reply:
Good point. The drag is essentially an external forcing.